### ISSUE TITLE
Support for CentOS 6

### ISSUE THREAD

**Issue by danderson** [tailscale/tailscale#315]
CentOS 6 is still supported, in the "maintenance updates only" phase of life according to https://en.wikipedia.org/wiki/CentOS#End-of-support_schedule . It EOLs in 6 months.

We don't ship Tailscale packages for CentOS 6 for a couple of reasons:
 - It doesn't run systemd, so we'd have to write and support legacy sysvinit scripts, and reimplement missing functionality like automatic restarts. CentOS 6 is the only distro in our current list that would need this.
 - It's missing policy routing features that we require to avoid routing loops (#144). (This is not implemented yet, but fixes a high-priority bug on linux)

In other words, CentOS 6 would be a special singleton compared to other distros we support. Given that it's reaching EOL in 6 months, it doesn't seem worth supporting at this point. This issue exists as something we can point to, and track whether to revisit this choice.

**Comment by DentonGentry** [comment]
As something to point to, I think this issue works as well being Closed as it does being Open.
